Source: Gentoo
Upstream: Unknown
Reason: Fix building with developmental glib/wxGTK, GSocket conflict.
--- audacity-src-1.3.8-orig/src/AudacityApp.cpp
+++ audacity-src-1.3.8/src/AudacityApp.cpp
@@ -331,7 +331,9 @@
 ///////////////////////////////////////////////////////////////////////////////
 
 #include <dlfcn.h>
-#include <gtk/gtk.h>
+/*#include <gtk/gtk.h>*/
+#include <glib/gtypes.h>
+#include <glib-object.h>
 
 typedef struct _GnomeProgram GnomeProgram;
 typedef struct _GnomeModuleInfo GnomeModuleInfo;
